core/src/test/java/org/infinispan/stream/DistributedStreamIteratorTest.java

```diff
@@ -296,7 +296,7 @@ public void testNodeLeavesWhileIteratingOverContainerCausingRehashToLoseValues()
       final BlockingQueue<Map.Entry<Object, String>> returnQueue = new LinkedBlockingQueue<>();
       Future<Void> future = fork(() -> {
          // Put batch size to a lower number just to make sure it doesn't retrieve them all in 1 go
-         Iterator<Map.Entry<Object, String>> iter = cache2.entrySet().stream().distributedBatchSize(2).iterator();
+         Iterator<Map.Entry<Object, String>> iter = cache2.entrySet().stream().distributedBatchSize(cache2.size() - 2).iterator();
```

Comment:
Is this change required?

Reply:
No, it was an attempt to reproduce before I had the new test.